pymess/backend/emails/__init__.py

```diff
@@ -67,30 +67,6 @@ def create_message(self, sender, sender_name, recipient, subject, content, relat
         except PersistenceException as ex:
             raise self.EmailSendingError(str(ex))
 
-    def send(self, sender, recipient, subject, content, sender_name=None, related_objects=None, tag=None,
```

Comment:
Not needed, it just added a bunch of extra arguments, by they are also defined in `create_message`, which is enough.